static/sass/_pattern_icons.scss

```diff
@@ -6,23 +6,23 @@ $color-icons-background: #999;
 @mixin canonical-p-icons {
   .is-dark {
     @include canonical-p-icon-facebook;
-    @include canonical-p-icon-twitter;
     @include canonical-p-icon-youtube;
+    @include canonical-p-icon-twitter;
   }
 }
 
 @mixin canonical-icon-facebook($background, $foreground) {
```

Comment:
You don't need to re-implement these mixins, you can use the ones from the core pattern, which now take foreground and background colours